ui/src/components/Main.js

```diff
@@ -4,6 +4,7 @@ import { Redirect, Match } from "react-router";
 import { Loader, Dimmer } from "semantic-ui-react";
 
 import Navigation from "./layout/Navigation";
+import Layout from "./layout/Layout.css";
```

Comment:
This can just be:

```
import "./layout/Layout.css";
```

That should get rid of the warning about `Layout` not being used.